frontend/integration-tests/tests/add-secrets.scenario.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable max-nested-callbacks */
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import { execSync } from 'child_process';
+import * as secretsView from '../views/secrets.view';
+import * as addSecretView from '../views/modal-add-secret-to-workload.view';
+import { searchJSON } from '../views/modal-add-secret-to-workload.view';
+
+describe('Add Secret to Workloads', () => {
+  const secretName = 'test-secret';
+  const resourceName = 'example';
+  const resourceKind = 'deployment';
+  const envPrefix = 'env-';
+  const mountPath = '/tmp/testdata';
+
+
+  beforeAll(async()=> {
+    // create deployment and secret
+    execSync(`kubectl run ${resourceName} --image=aosqe/hello-openshift -n ${testName}`);
+    execSync(`kubectl create secret generic ${secretName} --from-literal=key1=supersecret -n ${testName}`);
+  });
+
+  beforeEach(async() => secretsView.visitOneSecretPage(appHost, testName, secretName));
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  describe('Add Secret to Workloads as Enviroment Variables', ()=> {
```

Comment:
Both `Add Secret to Workloads as Enviroment Variables` and `Add Secret to Workloads as Volume` contain similar steps. Rather then having almost the same code in both TC's would extract the logic into a function that could be reused in both TC's.

Reply:
ok, will update